R/summaries.R

```diff
@@ -258,11 +261,22 @@ setMethod("countDominantTaxa", signature = c(x = "SummarizedExperiment"),
                      call. = FALSE)
             }
         }
+        # name check
+        if(!is.null(name)) {
+          if(is.character(name)==FALSE) {
+            stop("'name' must be a character value", call. = FALSE)
+          }
+        }
```

Comment:
Remove this nested if --> 


    if(!.is_non_empty_string(name)){
        stop("'name' must be a non-empty single character value.",
             call. = FALSE)
    }